CodeEdit/Features/Tabs/Views/TabBarItemView.swift

```diff
@@ -112,7 +115,7 @@ struct TabBarItemView: View {
     func closeAction() {
         isAppeared = false
         withAnimation(
-            .easeOut(duration: prefs.preferences.general.tabBarStyle == .native ? 0.15 : 0.20)
+            .easeOut(duration: tabBarStyle == .native ? 0.15 : 0.20)
```

Comment:
There are a lot of magic numbers here. Is it possible to do something about that?